### ISSUE TITLE
accessing _cachedsystemanimationfence requires the main thread ios

### ISSUE THREAD

**Issue by ghensley** [BlinkID/blinkid-react-native-ios#3]
Was occasionally getting this error when loading the scanning screen. Forcing dispatch from the main thread was a fix for me, but wondering if others have run into the same issue.

```
dispatch_sync(dispatch_get_main_queue(), ^{
    [rootViewController presentViewController:scanningViewController animated:YES completion:nil];
});
```

**Comment by intii** [comment]
+1. 
Happens for example when trying to trigger the scanning screen after interacting with a `<TextInput />`. Forcing dispatch from the main thread fixes the issue 👍

**Comment by Cerovec** [comment]
@intii, @ghensley - this should be solved by merged PR #4 

Please verify so we can close it.

**Comment by ghensley** [comment]
That's the change I have live right now :)
